storage_service/locations/forms.py

```diff
@@ -163,9 +163,11 @@ class Meta:
 
 
 class LocationForm(forms.ModelForm):
+    default = forms.BooleanField(required=False)
```

Comment:
Can this have a help string, e.g., like the "Enabled" attribute has "True if space can be accessed"? 

Speaking of which, it seems confusing that it's not "True if location can be accessed" at https://github.com/artefactual/archivematica-storage-service/blob/qa/0.x/storage_service/locations/models/location.py#L75, no?